### ISSUE TITLE
Can no longer log in to tidal-cli

### ISSUE THREAD

**Issue by brodel77** [okonek/tidal-cli-client#78]
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'test' ]
2 info using npm@3.5.2
3 info using node@v8.10.0
4 verbose config Skipping project config: /home/andrew/.npmrc. (matches userconfig)
5 verbose stack Error: ENOENT: no such file or directory, open '/home/andrew/package.json'
6 verbose cwd /home/andrew
7 error Linux 4.15.0-64-generic
8 error argv "/usr/bin/node" "/usr/bin/npm" "run" "test"
9 error node v8.10.0
10 error npm  v3.5.2
11 error path /home/andrew/package.json
12 error code ENOENT
13 error errno -2
14 error syscall open
15 error enoent ENOENT: no such file or directory, open '/home/andrew/package.json'
16 error enoent ENOENT: no such file or directory, open '/home/andrew/package.json'
16 error enoent This is most likely not a problem with npm itself
16 error enoent and is related to npm not being able to find a file.
17 verbose exit [ -2, true ]

**Comment by Medik-GH** [comment]
Same here, looks like Tidal updated their API.

**Comment by bigfreak85** [comment]
same here

**Comment by okonek** [comment]
Yes, the tidal has changed theire api. I am currently working on a completely new api library. Hope it will be ready soon!

**Comment by AdamStopa** [comment]
@okonek it is great to hear that!

**Comment by changemenemo** [comment]
no news about this problem?

**Comment by unmestplayer** [comment]
I am still having problems logging in, is anyone else experiencing this

**Comment by gitsaket** [comment]
the login is not working for the application.

**Comment by cdchris258** [comment]
any update or is this dead in the water

**Comment by unmestplayer** [comment]
> any update or is this dead in the water

You can use strawberry instead of this, you just sign in with your tidal details and you then get access to all you tracks.

**Comment by brendanrankin** [comment]
Dang it! Looks like this CLI/API is no longer functional! :-( I would have really liked to use it!

**Comment by brodel77** [comment]
> > any update or is this dead in the water
> 
> You can use strawberry instead of this, you just sign in with your tidal details and you then get access to all you tracks.

Thank you!

 I downloaded strawberry here: https://snapcraft.io/install/strawberry/mint#install

Then launched in terminal with sudo snap run strawberry

Entered my Tidal username/password and this API key kgsOOmYk3zShYrNP
